index.js

```diff
@@ -35,7 +35,7 @@ export default function(options, storage, key) {
   }
 
   function reducer(state, paths) {
-    return paths.length === 0
+    return paths === null || paths === undefined
```

Comment:
What about `!Array.isArray(path)` instead of checking both `null` and `undefined`?